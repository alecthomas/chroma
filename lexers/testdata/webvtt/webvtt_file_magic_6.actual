WEBVTT with more than two line breaks following





00:00.000 --> 00:01.000
Mock cue