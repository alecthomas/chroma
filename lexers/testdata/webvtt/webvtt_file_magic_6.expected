[
  {"type":"Keyword","value":"WEBVTT"},
  {"type":"Text","value":" with more than two line breaks following\n\n\n\n\n\n"},
  {"type":"LiteralDate","value":"00:00.000"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"--\u003e"},
  {"type":"Text","value":" "},
  {"type":"LiteralDate","value":"00:01.000"},
  {"type":"Text","value":"\nMock cue"}
]
